site/content/en/docs/enterprise/social-accounts-configuration.md

```diff
@@ -106,32 +106,72 @@ There are 2 basic steps to enable GitHub account authentication.
-
-To enable authentication, do the following:
-
-1. Create a user pool. For more information,
-   see [Amazon Cognito user pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
-2. Fill in the name field, set the homepage URL (for example: `https://localhost:8080`),
-   and authentication callback URL (for example: `https://localhost:8080/api/auth/social/amazon-cognito/login/callback/`).
-3. Create configuration file in CVAT:
-
-   1. Create the `auth_config.yml` file with the following content:
-
-   ```yaml
-   ---
-   social_account:
-     enabled: true
-     amazon_cognito:
-       client_id: <some_client_id>
-       client_secret: <some_client_secret>
-       domain: https://<domain-prefix>.auth.us-east-1.amazoncognito.com
-   ```
-
-   2. Set `AUTH_CONFIG_PATH="<path_to_auth_config>` environment variable.
-
-3. In a terminal, run the following command:
-
-   ```bash
-   docker compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.override.yml up -d --build
-   ```
+## Enable authentication with Amazon Cognito
+
+To enable authentication with Amazon Cognito for your CVAT instance, follow these steps:
+
+1. Create an **[Amazon Cognito pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)**
+   (_Optional_)
+1. Set up a new app client
+1. Configure social authentication in CVAT
+
+Now, let’s dive deeper into how to accomplish these steps.
+
+### Amazon Cognito pool creation
+
+This step is optional and should only be performed if a `user pool` has not already been created.
+To create a user pool, follow these instructions:
+1. Go to the [AWS Management Console](https://console.aws.amazon.com/console/home)
+1. Locate `Cognito` in the list of services
+1. Click `Create user pool`
+1. Fill in the required fields
+
+### App client creation
+
+To create a new `app client`, follow these steps:
+1. Go to the details page of the created user pool
+1. Find the `App clients` item in the menu on the left
+1. Click `Create app client`
+1. Fill out the form as shown bellow:
+   ![](/images/cognito_pool_1.png)
+   - `Application type`: `Traditional web application`
+   - `Application name`: Specify a desired name, or leave the autogenerated one
+   - `Return URL` (_optional_): Specify CVAT redirect URL.
+     (`<http|https>://<cvat_domain>/api/auth/social/amazon-cognito/login/callback/`).
+     This setting can also be updated or specified later after the app client is created.
+1. Navigate to the `Login pages` tab of the created app client
+1. Check the parameters in the `Managed login pages configuration` section and edit them if needed:
+   ![](/images/cognito_pool_2.png)
+   - `Allowed callback URLs`: Must be set to CVAT redirect URL
+   - `Identity providers`: Must be specified
+   - `OAuth grant types`: The `Authorization code grant` must be selected
+   - `OpenID Connect scopes`: `OpenID`, `Profile`, `Email` scopes must be selected
```

Comment:
```suggestion
   - `OpenID Connect scopes`: the `openid`, `profile`, `email` scopes must be selected
```

(they're lowercase in your screenshot)

Reply:
they are in uppercase when editing the settings
![image](https://github.com/user-attachments/assets/a7e11d32-b163-4939-8d20-51d5fabf0356)

Reply:
Alright.